contracts/RewardEscrowV2.sol

```diff
@@ -354,10 +364,12 @@ contract RewardEscrowV2 is
             // trigger event
             emit Vested(msg.sender, total);
 
-            // Send any fee to Treasury
+            // Send 50% any fee to Treasury and
+            // 50% to EarlyVestFeeDistributor
             if (totalFee != 0) {
                 /// @dev this will revert if the kwenta token transfer fails
-                kwenta.transfer(treasuryDAO, totalFee);
+                kwenta.transfer(treasuryDAO, totalFee * 50 / 100);
+                kwenta.transfer(earlyVestFeeDistributor, totalFee * 50 / 100);
```

Comment:
**Suggestion:** calculate `totalFee * 50 / 100` once and then pass it into both transfers.